bin/install_gems

```diff
@@ -6,6 +6,9 @@ GEMFILE_HASH=$(hash_file Gemfile.lock)
 CACHEKEY="$BUILDKITE_PIPELINE_SLUG-$ARCHITECTURE-ruby$RUBY_VERSION-$GEMFILE_HASH"
 
 restore_cache "$CACHEKEY"
+
+# Ensure we're using the latest version of Bundler before running bundle install
```

Comment:
Might be worth adding a note about why it is necessary to do this?

A link to https://github.com/Automattic/bash-cache-buildkite-plugin/issues/16 might be enough.

```suggestion
# Ensure we're using the latest version of Bundler before running bundle install.
#
# See https://github.com/Automattic/bash-cache-buildkite-plugin/issues/16  for more details on why this is recommended.
```